src/fsharp/FSharp.Compiler.Unittests/FscBuildLibrary.fs

```diff
@@ -0,0 +1,22 @@
+﻿namespace FSharp.Compiler.Unittests.Fsc
```

Comment:
If this file contains smoke tests, it should have a name representing that -- "FscSmokeTests.fs" or something. This is not a build library so "FscBuildLibrary" doesn't make sense to me.